sdk/sdk-api/src/main/java/mil/emp3/api/MilStdSymbol.java

```diff
@@ -1637,4 +1641,41 @@ public IEmpBoundingBox getFeatureBoundingBox() {
 
         return bBox;
     }
+
+    public void setIconColor(final IGeoColor color) {
+        this.attributes.put(MilStdAttributes.IconColor, ColorUtils.colorToString(color));
+    }
+
+    public void setFillColor(final IGeoColor color) {
+        this.attributes.put(MilStdAttributes.FillColor, ColorUtils.colorToString(color));
+    }
+
+    public void setLineColor(final IGeoColor color) {
+        this.attributes.put(MilStdAttributes.LineColor, ColorUtils.colorToString(color));
+    }
+
+    /**
+     * Convenience method to color fill, line and icon in one call.
+     * @param fillColor - Color of the fill.
+     * @param lineColor - Color of the line.
+     * @param iconColor - Color of the icon.
+     */
+    public void styleSymbol(final IGeoColor fillColor, final IGeoColor lineColor, final IGeoColor iconColor) {
+        this.setFillColor(fillColor);
+        this.setLineColor(lineColor);
+        this.setIconColor(iconColor);
+    }
+
+    public SparseArray<String> getAttributes() {
+        return this.attributes;
+    }
+
+    private void initializeDefaultAttributes() {
+        // TODO - The addition of attributes in the symbol causes an empty sparse array.
+        // TODO - Previously we had passed null as the attribute array which caused the renderer to use preset defaults
+        // TODO - when rendering. We can no longer do that as we need to use the array. Find these defaults and set them,
+        // TODO - The below are only guesses. I have had difficulty finding the actual defaults.
+        this.attributes.put(MilStdAttributes.Scale, "1");
+        this.attributes.put(MilStdAttributes.PixelSize, "150");
```

Comment:
I would suggest making a `DEFAULT_SCALE` and `DEFAULT_PIXEL_SIZE` static class variable so when we do fix them they are adequately labeled and dont need to be searched for.

Reply:
Done. Made them ints as in reality they are numbers but they are used as strings in the sparse array. Let me know if that should change.

Reply:
Im fine with them being `Int`s because I imagine you call a `toString` when you do the `put(...)`.

Reply:
Making them `Integer` and not `int` would be your approach.